src/Surfnet/StepupSelfService/SamlStepupProviderBundle/Saml/AssertionAdapter.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/**
+ * Copyright 2014 SURFnet bv
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\StepupSelfService\SamlStepupProviderBundle\Saml;
+
+use SAML2_Assertion as Assertion;
+
+class AssertionAdapter
```

Comment:
Might be better (seeing we implement this in a couple of places now) to push this functionality down to the [SamlBundle AssertionAdapter][1]

[1]: https://github.com/SURFnet/Stepup-saml-bundle/blob/b62b06fdde54086ffd68cbe056dd25e32ee88c67/src/SAML2/Response/AssertionAdapter.php#L27